modules/application/pages/index.adoc

```diff
@@ -8,7 +8,41 @@ You can easily analyze all execution data of your processes to better understand
 
 It is also possible to set time-based goals (Business Goals) to verify the achievement of your business objectives.
 
+== Prerequisites
+
+* Internet connexion
+
+* Any web browser
+
+* At least one functional process diagram and a BDM defined in Bonita Studio (version 2021.2 and beyond)
+
+* A Bonita Process Insights account / license
+
+* Be able to run the export tool on your BPM database
```

Comment:
access to your BPM database

you need to explain this flow for Bonita Cloud customers that don't have access to this database

Reply:
text modified